docs/guides/usage.md

```diff
@@ -0,0 +1,83 @@
+# How to use the lakeFS file system
+
+This guide contains instructions and code snippets on how to use the lakeFS file system.
+
+## Uploading and downloading files
+
+The arguably most important feature of the file system are file transfers.
+
+### File uploads
+
+To upload a file, you can use the `fs.put()` and `fs.put_file` methods:
```

Comment:
```suggestion
To upload a file, you can use the `fs.put()` and `fs.put_file()` methods:
```

Reply:
As a user, this paragraph doesn't make the difference between these two methods clear. Can you elaborate a bit?

Reply:
Added a sentence on the distinction in [8ca0a64](https://github.com/aai-institute/lakefs-spec/pull/156/commits/8ca0a64d1045e89f9db40f7f03c6f693f2f3891a).

Since the distinction is largely (or only?) the `recursive` switch, we could also decide to advertise the `get()`, `put()`, `copy()`, and `rm()` bulk operations - your call.

Reply:
I like the new version, let's keep it!